packages/next/build/webpack/plugins/profiling-plugin.ts

```diff
@@ -14,26 +19,85 @@ function getNormalModuleLoaderHook(compilation: any) {
   return compilation.hooks.normalModuleLoader
 }
 
+function tracingIsEnabled() {
```

Comment:
I found something that works! No need for a top-level span to be present for the plugin to activate.

Reply:
Should this be moved to tracer.ts so that it can be reused in other places if needed?